sources/packages/api/src/services/student/student.service.ts

```diff
@@ -303,17 +303,16 @@ export class StudentService extends RecordDataModelService<Student> {
   }
 
   /**
-   * Search the student based on the search criteria.
-   * @param firstName firsName of the student.
-   * @param lastName lastName of the student.
-   * @param appNumber application number of the student.
-   * @returns Searched student details.
+   * Search students based on the search criteria.
+   * @param searchCriteria options to search by firstName,
+   * lastName or appNumber.
+   * @returns searched student details.
```

Comment:
is it student details or list of students

Reply:
It is a subset with firstName, lastName, birthDate. What is the point here?

Reply:
mm. I see that promise is a student array, so it should be a list of student's detail right?

Reply:
Changed